TaylorFramework/Modules/temper/Core/Temper.swift

```diff
@@ -106,11 +106,12 @@ final class Temper {
      */
     
     func setLimits(limits: [String:Int]) {
-        rules = rules.map({ (var rule: Rule) -> Rule in
+        rules = rules.map({ ( rule: Rule) -> Rule in
+            var ruleCopy = rule
```

Comment:
I believe `mutableRule` would make a better name here because it would give more insights  on why it's used.